index.js

```diff
@@ -6,9 +6,10 @@ import { connect } from './lib/connected-component';
 import { createComponent } from './lib/observable-component';
 import { Component, observer } from './lib/observer-component';
 import * as propertyDecorators from './property-decorators';
+import makeObserve from 'can-observe/src/-make-observe';
+import React from 'react';
```

Comment:
I don't think we want React as a hard dep right?

Reply:
We import it into other files in the library too.

It is a peer dependency though